ui/apps/platform/src/Containers/Vulnerabilities/utils/searchUtils.tsx

```diff
@@ -217,6 +217,31 @@ export function getVulnStateScopedQueryString(
     });
 }
 
+export function getZeroCveScopedQueryString(
+    searchFilter: QuerySearchFilter,
+    currentEntity: WorkloadEntityTab
+): string {
+    let zeroCveFilterField: string;
+    switch (currentEntity) {
+        case 'Image':
+            zeroCveFilterField = 'Image CVE Count';
+            break;
+        case 'Deployment':
+            zeroCveFilterField = 'Deployment CVE Count';
```

Comment:
These two fields are yet-to-be implemented by the backend. Enabling the feature flag for the zero CVEs view depends both on completion of the UI work and implementation of these search fields.